src/smc-webapp/projects.cjsx

```diff
@@ -502,7 +502,7 @@ class ProjectsStore extends Store
                     return pay
 
     is_deleted: (project_id) =>
-        return !!@getIn(['project_map', project_id, 'deleted'])
+        return !!@getIn(['project_map', project_id, 'deleted']) #getIn returns nested data form list pointers. e.g. 'project_map' points to project_id, which points to 'deleted. 
```

Comment:
In the comment, `‘deleted` has an opening quote but no quote.  Also, I find the comment hard to read.  Finally, it makes no sense to explain what getIn is in this *one* place.   It’s used in hundred of places; it’s better to not have a comment here, and instead have a comment in some dev guide...?